### PR TITLE
Apply idea code analysis suggestions P-Security

### PR DESCRIPTION
**Pull request by mocenas**
### Summary

Apply idea code analysis tips to modules P-Security. Mostly updating deprecated API usages, removing unused things and code simplification.

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [X] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by mocenas** [comment]
run tests

**Comment by mocenas** [comment]
OCP failure is in a module which is not changed in this PR. Also seems to be just a fluke with image pulling.
So failure unrelated.

File: security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/services/AbstractRedisDao.java [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
 import io.smallrye.mutiny.Multi;
 import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.redis.client.RedisAPI;
 import io.vertx.mutiny.redis.client.Response;
 
 public abstract class AbstractRedisDao<E extends Record> {
 
     @Inject
-    ReactiveRedisClient redisClient;
+    RedisAPI redisClient;
```
Comment (gtroitsk) [inline security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/services/AbstractRedisDao.java:23]: Is this equivalent change from reactive perspective. Can you please confirm it?

File: properties/src/test/java/io/quarkus/ts/properties/toggle/BaseToggleablePropertiesIT.java [hunk 2]
```diff
@@ -36,16 +36,12 @@ private void whenEnableServiceAtRuntime(ToggleableServices service) {
     }
 
     private void thenServiceIsRunning(ToggleableServices service) {
-        Awaitility.await().atMost(Duration.ofSeconds(5)).untilAsserted(() -> {
-            given().get(service.getEndpoint())
-                    .then().statusCode(HttpStatus.SC_OK);
-        });
+        Awaitility.await().atMost(Duration.ofSeconds(5)).untilAsserted(() -> given().get(service.getEndpoint())
```
Comment (gtroitsk) [inline properties/src/test/java/io/quarkus/ts/properties/toggle/BaseToggleablePropertiesIT.java:39]: I think, when `given().get` part is on the next line it looks better

File: security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/services/AbstractRedisDao.java (hunk 1 above)
Comment (mocenas) [reply security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/services/AbstractRedisDao.java:23]: `RedisAPI` is implementing both [reactive and imperative API](https://quarkus.io/guides/redis-reference#apis). But imperative methods are suffixed with string like "andWait" (e.g. method `ftAliasaddAndAwait(...)` vs. reactive one `ftAliasadd(...)`).
So yes `RedisAPI` should be equivalent from reactive perspective.

Comment (gtroitsk) [reply security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/services/AbstractRedisDao.java:23]: Ok, thanks for explanation

**Review by gtroitsk — APPROVED** [review]
lgtm, thank you. OCP CI was triggered before and failures were unrelated. Let's wait for GH CI and merge

**Pull request merged**